### PR TITLE
partners/unstructured: fix client api_url

### PR DESCRIPTION
**Pull request by Coniferish**
**Description:** Add empty string default for api_key  and change `server_url` to `url` to match existing loaders.

- [x] **Lint and test**: Run `make format`, `make lint` and `make test` from the root of the package(s) you've modified. See contribution guidelines for more: https://python.langchain.com/docs/contributing/

Additional guidelines:
- Make sure optional dependencies are imported within a function.
- Please do not add dependencies to pyproject.toml files (even optional ones) unless they are required for unit tests.
- Most PRs should not touch more than one package.
- Changes should be backwards compatible.
- If you are adding something to community, do not re-import it in langchain.

If no one reviews your PR within a few days, please @-mention one of baskaryan, efriis, eyurtsev, ccurme, vbarda, hwchase17.

### CODE REVIEW

File: libs/partners/unstructured/langchain_unstructured/document_loaders.py [hunk 1]
```diff
@@ -106,17 +106,17 @@ def __init__(
...
                 )
 
-        unstructured_api_key = api_key or os.getenv("UNSTRUCTURED_API_KEY")
-        unstructured_url = server_url or os.getenv("UNSTRUCTURED_URL") or _DEFAULT_URL
+        self.file_path = file_path
+        self.file = file
+        self.partition_via_api = partition_via_api
+        self.post_processors = post_processors
 
+        unstructured_api_key = api_key or os.getenv("UNSTRUCTURED_API_KEY") or ""
```
Comment (efriis) [inline libs/partners/unstructured/langchain_unstructured/document_loaders.py:114]: yes on this line!

**Review by efriis — COMMENTED** [review]
can we just do line 114 and revert the rest?

**Comment by Coniferish** [comment]
> can we just do line 114 and revert the rest?

I'd like to keep the switch to `url` so users are referencing the same param as the other, preexisting loaders, but I'll revert rearranging the variable orders

File: libs/partners/unstructured/langchain_unstructured/document_loaders.py [hunk 2]
```diff
@@ -88,14 +88,14 @@ def __init__(
         # SDK parameters
         api_key: Optional[str] = None,
         client: Optional[UnstructuredClient] = None,
-        server_url: Optional[str] = None,
+        url: Optional[str] = None,
```
Comment (baskaryan) [inline libs/partners/unstructured/langchain_unstructured/document_loaders.py:91]: breaking change

Comment (efriis) [reply libs/partners/unstructured/langchain_unstructured/document_loaders.py]: I'm ok with this as a 0.1.1 release, and a yank of 0.1.0 - this is currently an unannounced package.

Comment (Coniferish) [reply libs/partners/unstructured/langchain_unstructured/document_loaders.py:91]: I dropped the commit that changed it back to `server_url` and updated the version to 0.1.1, so the PR should be good to go now @efriis

**Review by efriis — APPROVED** [review]

**Pull request merged**